opsdroid/connector/github/__init__.py

```diff
@@ -113,7 +113,7 @@ async def handle_check_event(self, payload: dict, user: str) -> github_events:
                 sender=user,
             )
 
-        if payload["action"] == "completed":
+        elif payload["action"] == "completed":
```

Comment:
Good catch. I did think it was weird that they were all indepedent `if` statements but didn't think about that might cause problems.